sql/admin.md

```diff
@@ -143,6 +143,44 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...
 
 用于查看当前 DDL 作业队列中的所有结果（包括正在运行以及等待运行的任务）以及已执行完成的 DDL 作业队列中的最近十条结果。
 
+  ```sql
+  mysql> admin show ddl jobs;
+  +--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
+  | JOB_ID | DB_NAME | TABLE_NAME | JOB_TYPE      | SCHEMA_STATE         | SCHEMA_ID | TABLE_ID | ROW_COUNT | START_TIME                        | STATE         |
+  +--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
+  | 45     | test    | t1         | add index     | write reorganization | 32        | 37       | 0         | 2019-01-10 12:38:36.501 +0800 CST | running       |
+  | 44     | test    | t1         | add index     | none                 | 32        | 37       | 0         | 2019-01-10 12:36:55.18 +0800 CST  | rollback done |
+  | 43     | test    | t1         | add index     | public               | 32        | 37       | 6         | 2019-01-10 12:35:13.66 +0800 CST  | synced        |
+  | 42     | test    | t1         | drop index    | none                 | 32        | 37       | 0         | 2019-01-10 12:34:35.204 +0800 CST | synced        |
+  | 41     | test    | t1         | add index     | public               | 32        | 37       | 0         | 2019-01-10 12:33:22.62 +0800 CST  | synced        |
+  | 40     | test    | t1         | drop column   | none                 | 32        | 37       | 0         | 2019-01-10 12:33:08.212 +0800 CST | synced        |
+  | 39     | test    | t1         | add column    | public               | 32        | 37       | 0         | 2019-01-10 12:32:55.42 +0800 CST  | synced        |
+  | 38     | test    | t1         | create table  | public               | 32        | 37       | 0         | 2019-01-10 12:32:41.956 +0800 CST | synced        |
+  | 36     | test    |            | drop table    | none                 | 32        | 34       | 0         | 2019-01-10 11:29:59.982 +0800 CST | synced        |
+  | 35     | test    |            | create table  | public               | 32        | 34       | 0         | 2019-01-10 11:29:40.741 +0800 CST | synced        |
+  | 33     | test    |            | create schema | public               | 32        | 0        | 0         | 2019-01-10 11:29:22.813 +0800 CST | synced        |
+  +--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
+  ```
+  * `JOB_ID`: 每个 DDL 操作对应一个DDL job, `JOB_ID` 全局唯一。
```

Comment:
- **Use Chinese punctuation in Chinese text.**
- **Do not leave a space both before and after Chinese punctuation.** 

Please check the following lines in this PR and update them. Thanks ^_^